osrdyne/src/drivers/worker_driver.rs

```diff
@@ -56,6 +60,13 @@ pub trait WorkerDriver: Send {
         worker_key: Key,
     ) -> Pin<Box<dyn Future<Output = Result<(), DriverError>> + Send + '_>>;
 
+    /// This methoid should check if current deployments are up to date and refresh them if needed.
```

Comment:
```suggestion
    /// This method should check if current deployments are up to date and refresh them if needed.
```
see https://www.merriam-webster.com/dictionary/methoid